modules/apps/account/account-api/src/main/java/com/liferay/account/service/result/ServiceResults.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.account.service.result;
+
+import java.util.List;
+
+/**
+ * @author Drew Brokke
+ */
+public class ServiceResults<T> {
```

Comment:
Just double checking on the naming. In the jira ticket description, you have `ServiceSearchResult` but `ServiceResults` in the diagram. Which name are we going to use?

Reply:
I'll try for `ServiceResults` first, since it's more generic. I assume Brian will have some input as well.